starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/substitutions.rocker.raw

```diff
@@ -0,0 +1,12 @@
+@import io.micronaut.starter.build.dependencies.Substitution
+@import java.util.Set
+@args (Set<Substitution> substitutions)
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+@for (substitution : substitutions) {
```

Comment:
What if `substitutions` is null or empty?

Reply:
I have wrapped it in an if statement